src/test/java/EmployeeTest.java

```diff
@@ -0,0 +1,151 @@
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.assertj.core.api.SoftAssertions;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.json.JSONObject;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EmployeeTest {
+
+    RequestSpecification requestSpecification;
+
+    @BeforeClass
+    public void createRequestSpecification() {
+        requestSpecification = new RequestSpecBuilder().
+                setBaseUri("http://dummy.restapiexample.com/api/v1").
+                build();
+    }
+
+    @DataProvider(name = "existingEmployees")
+    public Object[][] existingEmployees() {
+        return new Object[][]{
+                {1, "Tiger Nixon", 320800, 61, ""},
+                {2, "Garrett Winters", 170750, 63, ""},
+                {3, "Ashton Cox", 86000, 66, ""},
+                {4, "Cedric Kelly", 433060, 22, ""},
+                {5, "Airi Satou", 162700, 33, ""}
+        };
+    }
+
+    @Test(dataProvider = "existingEmployees")
+    public void checkGetEmployeesByIds(int id, String employeeName, int employeeSalary, int employeeAge, String profileImage) {
+        Response employeeByIdResponse = getEmployeeByIdResponse(id);
+
+        assertThat(employeeByIdResponse.statusCode()).as("check response status code").isEqualTo(200);
+
+        Employee employee = employeeByIdResponse.as(Employee.class);
+
+        SoftAssertions softly = new SoftAssertions();
+            softly.assertThat(employee.getData().getEmployee_name()).isEqualTo(employeeName);
+            softly.assertThat(employee.getData().getEmployee_salary()).isEqualTo(employeeSalary);
+            softly.assertThat(employee.getData().getEmployee_age()).isEqualTo(employeeAge);
+            softly.assertThat(employee.getData().getProfile_image()).isEqualTo(profileImage);
+        softly.assertAll();
+    }
+
+    @Test
+    public void checkForEmployeeWithNotExistingId() {
+        int notExistingId = 25;
+        Response employeeByIdResponse = getEmployeeByIdResponse(notExistingId);
+
+        assertThat(employeeByIdResponse.statusCode()).as("check response status code").isEqualTo(200);
+
+        Employee employee = employeeByIdResponse.as(Employee.class);
+
+        SoftAssertions softly = new SoftAssertions();
+            softly.assertThat(employee.getData()).isNull();
+            softly.assertThat(employee.getStatus()).isEqualTo("failure");
+            softly.assertThat(employee.getMessage()).isEqualTo("Failure! There is no such employee.");
+        softly.assertAll();
+    }
+
+    @Test
+    public void checkForNegativeAgeValueOfEmployees() {
```

Comment:
please remove this test, it does not make much sense IMHO. It would make sense to validate this during employee creation. But not here. Same for the other negative tests below.